src/util/colors.ts

```diff
@@ -8,14 +8,15 @@ import { IPartialTheme } from 'office-ui-fabric-react/lib/Styling';
 /* Blue */
 // const blue = '#2477c3';
 // const stream = '#e9eff8';
-// const pond = '#9ac3ff';
+const pond = '#9ac3ff';
 const lake = '#6c98d9';
 const river = '#386cbb';
-// const bay = '#264f8c';
+const bay = '#264f8c';
 // const ocean = '#19345d';
+const newBlue = '#2f5c9f';
```

Comment:
I've posted a question in yammer to clarify with design if we need this new color.